main.py

```diff
@@ -337,6 +342,50 @@ def isPaused(self):
 	def isRunning(self):
 		return self._p is not None # FIXME: Check subproc state
 
+class AppVeyor(object):
+	def __init__(self, repo, branch, user):
```

Comment:
I'd suggest `user, repo, branch` (which is common order in URL and makes most sense):

`user` has `repo`, which has `branch`.

Reply:
sure, seems a lot better, i will change that.